cookbook/configuration/index.rst

```diff
@@ -8,6 +8,8 @@ Configuration
     override_dir_structure
     front_controllers_and_kernel
     external_parameters
-    pdo_session_storage
+    database_session_storage
+    																																																																							
     apache_router
     web_server_configuration
+    cookbook/configuration/pdo_session_storage cookbook/configuration/database_session_storage
```

Comment:
this line should be moved to the ``redirection_map`` file in the root of this repository